TicTacToe.Test/BoardTest.cs

```diff
@@ -0,0 +1,157 @@
+﻿using System;
+using Xunit;
+
+namespace TicTacToe.Test
+{
+    public class BoardTest
+    {
+        [Fact]
+        public void SpecifyGameBoardSIze()
+        {
+            Board board = new Board(3);
```

Comment:
Since the set up for each test is the some, consider move this line to a setup method or constructor for XUnit.

Reply:
okay